src/Symfony/Component/Config/Definition/EnumNode.php

```diff
@@ -21,13 +21,19 @@
 class EnumNode extends ScalarNode
 {
     private array $values;
+    private ?string $enumFqcn = null;
```

Comment:
Naming things is hard. 😉

`$enumClass`, `$enumFqcn`: Let's find _one_ name for this variable and use that name consistently.